v0/destinations/klaviyo/transform.js

```diff
@@ -298,6 +271,32 @@ const groupRequestHandler = (message, category, destination) => {
   membersResponse.method = defaultPostRequestConfig.requestMethod;
   membersResponse.params = { api_key: destination.Config.privateApiKey };
   responseArray.push(membersResponse);
+
+  const targetUrl = `${BASE_ENDPOINT}/api/v2/list/${get(
+    message,
+    "groupId"
+  )}/subscribe`;
+  if (get(message.traits, "subscribe") === true) {
+    // Adding Consent Info to Profiles
+    let subscribeProfile =JSON.parse(JSON.stringify(profile));
+    subscribeProfile.sms_consent =
+      message?.context?.traits.smsConsent || destination.Config.smsConsent;
+    subscribeProfile.$consent =
+      message?.context?.traits.consent || destination.Config.consent;
```

Comment:
```suggestion
    subscribeProfile.sms_consent =
      message.context?.traits.smsConsent || destination.Config.smsConsent;
    subscribeProfile.$consent =
      message.context?.traits.consent || destination.Config.consent;
```

first level optional chaining is not required

Reply:
Done